guestbook-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -29,12 +29,6 @@
         <include>**/*.java</include>
       </includes>
     </fileSet>
-    <fileSet encoding="UTF-8">
-      <directory>eclipse-launch-profiles</directory>
-      <includes>
-        <include>**/*.launch</include>
-      </includes>
-    </fileSet>
```

Comment:
It seemed like a good idea since folks weren't using the eclipse plugin.  Ludo pointed out that these help Eclipse on MVN, so it will come back.